### PR TITLE
Bump component versions

### PR DESCRIPTION
**Pull request by simonpasquier**
* [ ] I added CHANGELOG entry for this change.
* [X] No user facing changes, so no entry in CHANGELOG was needed.

Relates to #847

### CODE REVIEW

**Comment by simonpasquier** [comment]
/hold
lets wait for https://github.com/openshift/prometheus/pull/52 to be merged

**Comment by paulfantom** [comment]
Images are overridden by CVO so in the context of OSE (and CI) it doesn't matter if we wait for https://github.com/openshift/prometheus/pull/52 or not. We need versions for 

/lgtm

**Comment by paulfantom** [comment]
/unhold

https://github.com/openshift/prometheus/pull/52 is merged

**Comment by paulfantom** [comment]
/lgtm cancel

We are shipping kube-rbac-proxy 0.6.0 not 0.5.0

File: assets/prometheus-user-workload/prometheus.yaml [hunk 1]
```diff
@@ -115,5 +115,5 @@ spec:
       requests:
         cpu: 1m
         memory: 100Mi
-    version: latest
-  version: v2.19.2
+    version: v0.14.0
```
Comment (paulfantom) [inline assets/prometheus-user-workload/prometheus.yaml:118]: It says 0.14.0 but image references 0.10.0

File: jsonnet/main.jsonnet [hunk 2]
```diff
@@ -80,13 +80,13 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                  // Because we build OpenShift images separately to upstream,
                  // we have to ensure these versions exist before upgrading.
                  openshiftOauthProxy: 'latest',
-                 prometheus: 'v2.19.2',
+                 prometheus: 'v2.20.0',
                  alertmanager: 'v0.21.0',
                  nodeExporter: 'v1.0.1',
                  promLabelProxy: 'v0.1.0',
-                 kubeRbacProxy: 'v0.4.1',
+                 kubeRbacProxy: 'v0.5.0',
```
Comment (paulfantom) [inline jsonnet/main.jsonnet:87]: 0.6.0 as per https://github.com/openshift/kube-rbac-proxy/blob/master/VERSION

**Comment by simonpasquier** [comment]
> Images are overridden by CVO so in the context of OSE (and CI) it doesn't matter if we wait for openshift/prometheus#52 or not.

sure it was more from a theoritical point of view :)

**Pull request merged**